tests/backup_helper.go

```diff
@@ -3818,6 +3818,22 @@ func GetPxBackupBuildDate() (string, error) {
 	return backupVersion.GetBuildDate(), nil
 }
 
+func IsCurrentPxBackupVersionGreaterThanOrEqualTo(v string) (bool, error) {
```

Comment:
Add docstring maybe here.

Reply:
Not needed. I will remove this function. I am using the one below this